### DISCUSSION TITLE
Taking all questions here

### DISCUSSION THREAD

**Discussion by benmoose39** [benmoose39/YouTube_to_m3u#3] [General]
Ask away your questions here or on the discord server: https://discord.gg/dmgYmAEdee

**Comment by weareblahs** [comment]
Is there any way to force resolution for YouTube streams?

**Reply by benmoose39** [reply]
Yes.
If you perform `requests.get()` on the m3u that you get from the script, you can find different links corresponding to different bandwidths.

**Reply by weareblahs** [reply]
> Yes.
> If you perform `requests.get()` on the m3u that you get from the script, you can find different links corresponding to different bandwidths.

Anyways, how to change the value of `requests.get()` to just get the 1080p stream?

**Reply by benmoose39** [reply]
can you dm on discord?
Some tweaks in the script and it should be working!

**Reply by weareblahs** [reply]
The Discord shows "Invalid Invite". Anyway, what's your Discord username so I can contact you about it?

**Reply by benmoose39** [reply]
I've updated the invite link.
